src/functions.php

```diff
@@ -27,7 +27,7 @@ function register_file($path)
 function console_log($expression, $_ = null)
 {
     ob_start();
-    call_user_func_array('var_dump', func_get_args());
+    var_dump(...func_get_args());
```

Comment:
Just until now we did support php 5.6

Reply:
It's 5.6 feature: http://php.net/manual/en/migration56.new-features.php

Reply:
Cool- I didn't know that!